`ga` should be in dependency array in tracking missing translations

Warning:
```
/Users/peterbe/dev/MOZILLA/MDN/kuma/kuma/javascript/src/article.jsx
  64:8  warning  React Hook useEffect has a missing dependency: 'ga'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

```

Something we missed when we landed 
https://github.com/mozilla/kuma/pull/5847

--- a/kuma/javascript/src/article.jsx
+++ b/kuma/javascript/src/article.jsx
@@ -61,7 +61,7 @@ export default function Article({ document }: DocumentProps) {
                 eventLabel: ''
             });
         }
-    }, [document, locale]);
+    }, [document, locale, ga]);
 
     return (
         /*

